apps/greenary-care-map/src/components/Detail/index.tsx

```diff
@@ -109,7 +108,7 @@ export const Detail = ({ features, onClose, isMobile, arcgisServerUrl }: DetailP
     </div>
   );
 
-  return isMobile ? (
+  return !isMobile ? (
```

Comment:
is this on purpose ? or forgotten from debug

Reply:
purpose, its strange created in oposite condition, i think it need more general refactor later this detail component, there never could work mobile detail